proto/c1/connector/v2/annotation_sync_id.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package c1.connector.v2;
+
+option go_package = "github.com/conductorone/baton-sdk/pb/c1/connector/v2";
+
+// SyncDetails is an annotation is used to communicate with the connector store, often for a _previous_ sync.
+message SyncId {
```

Comment:
Can we call this message ActiveSyncId so that there's no chance it gets mixed up with SyncDetails in the c1z protos?